### DISCUSSION TITLE
MCS%40UIUC/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#150] [Announcements]
# MCS%40UIUC/

https://csmsapp.org/MCS%40UIUC/

**Comment by ZhuoruLi** [comment]
建议重新评估

**Reply by xichenpan** [reply]
今年大家都冲，又变难了

**Reply by ZhuoruLi** [reply]
不会又成陆本黑洞了吧

**Reply by xichenpan** [reply]
maybe

**Reply by xichenpan** [reply]
也可能去年招了太多陆本，去了之后feedback不好吧

**Reply by ZhuoruLi** [reply]
呜呜呜

**Reply by RainJamesY** [reply]
24Fall 进一步黑洞了 陆本rej了一大批bg很好的选手，美本3.5一段科研没什么产出都能录

**Comment by HUANGLIZI** [comment]
主要是本校保研的也变多了

**Reply by ErwinZhou** [reply]
想问下UIUC的MSC的2024FALL录取结果，还是黑洞吗

**Comment by justinccu** [comment]
現在貌似可以defer最多到一年
只是要先繳750佔位費

**Comment by ciel307** [comment]
体感26fall大扩招降bar，或许是fund减少的原因。每一年情况都不太一样
